when i run the program with spark-submit i get errors

Exception in thread "main" java.lang.NoClassDefFoundError: org/apache/spark/streaming/twitter/TwitterUtils$
	at me.baghino.spark.intro.TwitterExampleBase$class.$init$(TwitterExampleBase.scala:32)
	at me.baghino.spark.intro.TwitterExample$.<init>(TwitterExample.scala:10)
	at me.baghino.spark.intro.TwitterExample$.<clinit>(TwitterExample.scala)
	at me.baghino.spark.intro.TwitterExample.main(TwitterExample.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.spark.deploy.SparkSubmit$.org$apache$spark$deploy$SparkSubmit$$runMain(SparkSubmit.scala:729)
	at org.apache.spark.deploy.SparkSubmit$.doRunMain$1(SparkSubmit.scala:185)
	at org.apache.spark.deploy.SparkSubmit$.submit(SparkSubmit.scala:210)
	at org.apache.spark.deploy.SparkSubmit$.main(SparkSubmit.scala:124)
	at org.apache.spark.deploy.SparkSubmit.main(SparkSubmit.scala)
Caused by: java.lang.ClassNotFoundException: org.apache.spark.streaming.twitter.TwitterUtils$
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 13 more

--- a/README.md
+++ b/README.md
@@ -14,4 +14,7 @@ Visit [apps.twitter.com](https://apps.twitter.com) to get your own API keys.
 To submit the job to an existing Spark installation you can run the following
 
     sbt assembly
-    $SPARK_HOME/bin/spark-submit --class me.baghino.spark.intro.TwitterExample --master $SPARK_MASTER target/scala-2.11/spark-twitter-stream-example-assembly-0.0.1.jar
+    $SPARK_HOME/bin/spark-submit \
+      --class me.baghino.spark.intro.TwitterExample \
+      --master $SPARK_MASTER \
+      target/scala-2.11/spark-twitter-stream-example-assembly-0.0.1.jar

--- a/README.md
+++ b/README.md
@@ -11,9 +11,12 @@ To make it work on your machine, be sure to add a `twitter4j.properties` under `
 
 Visit [apps.twitter.com](https://apps.twitter.com) to get your own API keys.
 
-To submit the job to an existing Spark installation you can run the following
+To submit the job to an existing Spark installation you can package the job with the following command
 
     sbt assembly
+
+and then submit it
+
     $SPARK_HOME/bin/spark-submit \
       --class me.baghino.spark.intro.TwitterExample \
       --master $SPARK_MASTER \

--- a/README.md
+++ b/README.md
@@ -10,3 +10,8 @@ To make it work on your machine, be sure to add a `twitter4j.properties` under `
     oauth.accessTokenSecret=***
 
 Visit [apps.twitter.com](https://apps.twitter.com) to get your own API keys.
+
+To submit the job to an existing Spark installation you can run the following
+
+    sbt assembly
+    $SPARK_HOME/bin/spark-submit --class me.baghino.spark.intro.TwitterExample --master $SPARK_MASTER target/scala-2.11/spark-twitter-stream-example-assembly-0.0.1.jar

--- a/build.sbt
+++ b/build.sbt
@@ -8,3 +8,14 @@ libraryDependencies ++= Seq(
   "org.apache.spark" % "spark-streaming_2.11" % "1.2.1",
   "org.apache.spark" % "spark-streaming-twitter_2.11" % "1.2.1"
 )
+
+assemblyMergeStrategy in assembly := {
+  case PathList("javax", "activation", xs @ _*)                   => MergeStrategy.first
+  case PathList("com", "esotericsoftware", xs @ _*)               => MergeStrategy.first
+  case PathList("org", "apache", "hadoop", xs @ _*)               => MergeStrategy.first
+  case PathList("org", "apache", "commons", xs @ _*)              => MergeStrategy.first
+  case PathList("org", "slf4j", xs @ _*)                          => MergeStrategy.first
+  case PathList(ps @ _*) if ps.last endsWith "plugin.properties"  => MergeStrategy.first
+  case PathList("META-INF", xs @ _*)                              => MergeStrategy.first
+  case x                                                          => (assemblyMergeStrategy in assembly).value.apply(x)
+}

--- a/project/plugins.sbt
+++ b/project/plugins.sbt
@@ -1,0 +1,1 @@
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.3")

